selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java

```diff
@@ -78,14 +85,26 @@ public void checkButtonsOnToolbar() {
     projectExplorer.waitProjectExplorer();
     String currentWindow = seleniumWebDriver.getWindowHandle();
     commandsToolbar.clickExecStopBtn();
+
+    String previewUrl = consoles.getPreviewUrl();
+    String expectedText =
+        previewUrl.contains("route")
+            ? "Application is not available"
```

Comment:
Could you, please, comment here why we depend on the preview url in such a strange manner?

Reply:
Also, it needs when the test is running on che6-ocp platform

Reply:
I see. Please, also add comment into the code to make it clear for other developers.